Client/Ecosia/Analytics/Analytics.Values.swift

```diff
@@ -145,7 +142,10 @@ extension Analytics {
             search,
             profits,
             action,
-            privacy
+            privacy,
+            // TODO: Review .greenSearch and .transparentFinances analytics values
```

Comment:
Is this `// TODO` regarding the possibility of splitting the analytics depending on the onboarding page shown?

Reply:
It is mostly just to remind that this new page names need to be approved by BI, still pending to check with them

Reply:
I guess we are good to go now 💪